src/__tests__/controllers/exampleData.json

```diff
@@ -0,0 +1,33 @@
+{
+  "volunteer": {
+    "firstName": "Halit",
+    "lastName": "Batur",
+    "birthDate": "01/01/2023",
+    "email": "halit@recoded.com"
+  },
+  "organization": {
+    "name": "Recoded",
+    "description": "Best place get tortured",
```

Comment:
No no we can't be that bad. 😆 
Let's change this to a more professional text, please?